fastlane/lanes/build.rb

```diff
@@ -0,0 +1,427 @@
+
+    comment_on_pr(
+      project: 'wordpress-mobile/wordpress-ios',
+      pr_number: Integer(ENV['BUILDKITE_PULL_REQUEST']),
+      reuse_identifier: 'installable-build-link',
+      body: comment_body
+    )
+  end
+
+  # Builds the Jetpack app for an Installable Build ("Jetpack" scheme), and uploads it to AppCenter
+  #
+  # @called_by CI
+  #
+  desc 'Builds and uploads a Jetpack installable build'
+  lane :build_and_upload_jetpack_installable_build do
+    sentry_check_cli_installed
+
+    jetpack_alpha_code_signing
+
+    # Get the current build version, and update it if needed
+    version_config_path = File.join(PROJECT_ROOT_FOLDER, 'config', 'Version.internal.xcconfig')
+    versions = Xcodeproj::Config.new(File.new(version_config_path)).to_hash
+    build_number = generate_installable_build_number
+    UI.message("Updating build version to #{build_number}")
+    versions['VERSION_LONG'] = build_number
+    new_config = Xcodeproj::Config.new(versions)
+    new_config.save_as(Pathname.new(version_config_path))
+
+    gym(
+      scheme: 'Jetpack',
+      workspace: WORKSPACE_PATH,
+      export_method: 'enterprise',
+      configuration: 'Release-Alpha',
+      clean: true,
+      output_directory: BUILD_PRODUCTS_PATH,
+      output_name: 'Jetpack Alpha',
+      derived_data_path: DERIVED_DATA_PATH,
+      export_team_id: ENV['INT_EXPORT_TEAM_ID'],
+      export_options: { method: 'enterprise' }
+    )
+
+    appcenter_upload(
+      api_token: get_required_env('APPCENTER_API_TOKEN'),
+      owner_name: 'automattic',
+      owner_type: 'organization',
+      app_name: 'jetpack-installable-builds',
+      file: lane_context[SharedValues::IPA_OUTPUT_PATH],
+      dsym: lane_context[SharedValues::DSYM_OUTPUT_PATH],
+      destinations: 'Collaborators',
+      notify_testers: false
+    )
+
+    sentry_upload_dsym(
+      auth_token: get_required_env('SENTRY_AUTH_TOKEN'),
+      org_slug: 'a8c',
+      project_slug: 'jetpack-ios',
+      dsym_path: lane_context[SharedValues::DSYM_OUTPUT_PATH]
+    )
+
+    return if ENV['BUILDKITE_PULL_REQUEST'].nil?
+
+    download_url = Actions.lane_context[SharedValues::APPCENTER_DOWNLOAD_LINK]
+    UI.message("Successfully built and uploaded installable build here: #{download_url}")
+    install_url = 'https://install.appcenter.ms/orgs/automattic/apps/jetpack-installable-builds/'
+
+    comment_body = "You can test the <strong>Jetpack</strong> changes on this Pull Request by downloading it from AppCenter <a href='#{install_url}'>here</a> with build number: <code>#{build_number}</code>. IPA is available <a href='#{download_url}'>here</a>. If you need access to this, you can ask a maintainer to add you."
```

Comment:
Same suggestion as the WordPress lane.

```suggestion
    comment_body = "You can test the changes in <strong>Jetpack</strong> from this Pull Request by downloading it from App Center <a href='#{install_url}'>here</a> with build number: <code>#{build_number}</code>. IPA is available <a href='#{download_url}'>here</a>. If you need access to this, you can ask a maintainer to add you."
```

Actually, I wonder if it would be best to extract this in a little helper function. I'd say we'd always want the comments to have the same format, so using a template for them might be handy.

This is of course a nitpick, and something that could very well be a followup PR.

Reply:
Will DRY the comment text in a follow-up PR

Reply:
Done in https://github.com/wordpress-mobile/WordPress-iOS/pull/18192